source/CMakeLists.txt

```diff
@@ -51,6 +43,7 @@ include_directories(
 # end of everything (e.g. numerics/vectors.cc takes several minutes to
 # compile...)
 #
+set(CMAKE_INCLUDE_CURRENT_DIR true)
```

Comment:
... and everything exploded :sweat_smile: I need this toggle so that CMake automatically adds the current source and binary directory of every object target. Otherwise we don't find the expanded `.inst` files.